readme.txt

```diff
@@ -0,0 +1,67 @@
+=== Komoju Payments ===
+Contributors: degica
+Tags: WooCommerce,Payment Gateway,Komoju
+Requires at least: 5.3
+Tested up to: 5.4
+Stable tag: trunk
+Requires PHP: 7.2
+License: MIT
+License URI: https://directory.fsf.org/wiki/License:X11
+
+A WooCommerce plugin to receive payments with Komoju, Japan's leading provider for online and offline payments.
+
+== Description ==
+
+The Komoju Payments WooCommerce plugin allows customers to checkout with a variety of Japanese payment options, include Credit Card, Konbini, Webmoney and more.
+
+To use this plugin you must have a [Komoju account](https://komoju.com/).
+
+= Configuration Instructions =
+Login to Wordpress as Administrator 
+
+Click on the 'Plugins' menu on the left hand side.
+
+You should see the WooCommerce Komoju Gateway listed among the plugins list. Click the Activate Link for this gateway.
+Next you need to configure the Plugin in WooCommerce. To do so, from the left hand menu, select 'WooCommerce' and then 'Settings'.
+
+Then click 'Checkout' from the Top Menu.
+
+Click on the 'Komoju' Link just below the top tabbed menu.
+
+Click the Enable/Disable Box to enable this gateway.
+
+Enter your Komoju API credentials in this configuration page. Those data to set here are the ones defined in your Komoju dashboard. Ignore the "Webhook Secret Token" for now, as it will be filled out after your Komoju account has been configured.
```

Comment:
I think we should explicitly list the credentials here

Reply:
Good idea, I've explicitly mentioned them in 743ecbf